modules/src/ics03_connection/exported.rs

```diff
@@ -0,0 +1,113 @@
+use super::error;
+use crate::ics23_commitment::CommitmentPrefix;
+use anomaly::fail;
+use serde_derive::{Deserialize, Serialize};
+
+pub trait ConnectionI {
```

Comment:
`I` suffix for traits is not a common Rust practice in my experience. How about naming the trait `Connection` instead?

Ambiguities can always be resolver by renaming imports or using the qualified path to the trait.

Reply:
I just made an attempt to fix this and the other comments you had. Also aligned connection and channel a bit more.